tests/test_config.py

```diff
@@ -51,3 +51,126 @@ def test_no_ini_path(self):
 
         with self.assertRaises(ValueError):
             parser.parse_args([])
+
+
+class TestGetConfigValue(unittest.TestCase):
+    """Tests for caramel.config._get_config_value"""
+
+    def test_argument_only(self):
+        """Test that a value will be returned even without settings or an
+        env-variable"""
+        variable_value = "very important configuration detail"
```

Comment:
The other tests lead with declaring "variable_name", please keep them all in the same order of common first, it helps me diff the tests between each other.